CHANGELOG.md

```diff
@@ -13,6 +13,7 @@
 Breaking changes:
 
 - [task] `TaskService.getConfiguredTasks()` returns `Promise<TaskConfiguration[]>` instead of `TaskConfiguration[]` [#5777](https://github.com/theia-ide/theia/pull/5777)
+- [outline] added `OutlineViewTreeModel` for the outline view tree widget [#5687](https://github.com/theia-ide/theia/pull/5687)
```

Comment:
i don't think it is breaking, APIs were not changed, just more specific, common APIs are still there

Reply:
Yep, I will remove it from there.